apis/v1alpha1/nginxproxy_types.go

```diff
@@ -200,6 +201,6 @@ type AddressType string
 
 const (
 	// AddressTypeCIDR specifies that the address is a CIDR block.
-	// kubebuilder:validation:Pattern=`(\/([0-9]?[0-9]?[0-8]))$`
+	// kubebuilder:validation:Pattern=`^[\.a-zA-Z0-9::]*(\/([0-9]?[0-9]?[0-8]))$`
```

Comment:
This needs to allow `0-9` for the last one as well, for example `/119`.

I also don't think we need to specify `:` twice in here.